### ISSUE TITLE
Network Section is too big

### ISSUE THREAD

**Issue by schenklklopfer** [mohsenasm/swarm-dashboard#43]
Hello,

I am currently deploying massively Stacks to my new cluster.
Each stack brings its networks and the network section on the swarm-dashboard gets too big.
Just have a look on this screenshot.
![grafik](https://github.com/user-attachments/assets/f8e0643c-e7cf-454a-af89-446dc6763f0d)

Currently I have deployed about 30% of all my stacks.
If I am done I am afraid I cannot see any containers anymore.

Is there a option to completely disable the network section, or to collapse it?
Or maybe get a fixed width with a scroll bar?

**Comment by mohsenasm** [comment]
Hi,
Currently, there is no option to disable the network section. We can add it in the future.

As a quick workaround, you can fork the repo and add this line in the [index.html](https://github.com/mohsenasm/swarm-dashboard/tree/master/elm-client/client) (at the end of the `<head>` tag).
```html
<style>.networks { display: none; }</style>
```

**Comment by schenklklopfer** [comment]
Thanks!
It works with a mount at the moment.

But it'd be great if you merge my PR for this :)

**Comment by mohsenasm** [comment]
Thanks for the PR 🙏🏻, but I think adding this option to the server is better. So that we can prevent sending the network section (which is big in this case) and save the processing time too. Also, I noticed that `display: none;` removes the left border of the table.

I have added this option with `ENABLE_NETWORKS` (not `ENABLE_NETWORK`) in the master branch and the `mohsenasm/swarm-dashboard:master` docker image.
